python/composio/client/enums/_action.py

```diff
@@ -1,9 +1,24 @@
 """
 Action enums.
 """
+
 import typing as t
 
-from composio.client.enums.base import ACTIONS_CACHE, ActionData, _AnnotatedEnum, enum
+import typing_extensions as te
+
+from composio.client.enums.base import (
+    ACTIONS_CACHE,
+    ActionData,
+    _AnnotatedEnum,
+    add_action,
+    enum,
+)
+
+
+class SentinalObject:
+    """Sentinal object."""
+
+    sentinal = None
```

Comment:
What is this for?

Reply:
This is workaround for allowing the objects marked with `action` decorator to be used as composio actions